tests/it/spec.js

```diff
@@ -154,7 +154,8 @@ describe('GET for root route ', function () {
 
   it('responds to /start with wrong format', function testSlash (done) {
     const errorMessage = 'Response not ok.'
-    const successMessage = 'Please enter the command in correct format e.g. /planning-poker start or stop JIRA-1001'
+    const successMessage = 'Please enter the command in correct format ' +
+    'e.g. /planning-poker start or stop or status JIRA-1001'
```

Comment:
Not sure why linting didn't catch this, but this line should be indented two more spaces.